config/routes.rb

```diff
@@ -142,9 +142,13 @@
   get "/node/:id" => "nodes#show", :id => /\d+/, :as => :node
   get "/node/:id/history" => "old_nodes#index", :id => /\d+/, :as => :node_history
   resources :old_nodes, :path => "/node/:id/history", :id => /\d+/, :version => /\d+/, :param => :version, :only => :show
+
   get "/relation/:id" => "relations#show", :id => /\d+/, :as => :relation
+  get "/relation/:id/member" => "relation_members#show", :id => /\d+/, :as => :relation_member
```

Comment:
Should this (and the other route) be `/members` given that they load all members not just on?

Reply:
make sense, fixed.